Sources/WalletConnectPairing/PairingRequestsSubscriber.swift

```diff
@@ -2,34 +2,48 @@ import Foundation
 import Combine
 import WalletConnectUtils
 import WalletConnectNetworking
+import JSONRPC
 
 public class PairingRequestsSubscriber {
     private let networkingInteractor: NetworkInteracting
     private let pairingStorage: PairingStorage
     private var publishers = Set<AnyCancellable>()
+    private var registeredProtocolMethods = SetStore<String>(label: "com.walletconnect.sdk.pairing.registered_protocol_methods")
+    private let pairingProtocolMethods = ["\(PairingPingProtocolMethod().method)", "\(PairingDeleteProtocolMethod().method)"]
```

Comment:
@flypaper0 I think we could make this more safe 🤔 
any idea how we could return all protocol methods when they are structs? and ensure that when new methods appears the list is also updated.

Reply:
commited here